recursive_art.py

```diff
@@ -190,20 +135,14 @@ def generate_art(filename, x_size=350, y_size=350):
             x = remap_interval(i, 0, x_size, -1, 1)
             y = remap_interval(j, 0, y_size, -1, 1)
             pixels[i, j] = (
-                    color_map(evaluate_random_function(red_function, x, y)),
-                    color_map(evaluate_random_function(green_function, x, y)),
-                    color_map(evaluate_random_function(blue_function, x, y))
+                    color_map(red_function(x, y)),
+                    color_map(green_function(x, y)),
+                    color_map(blue_function(x, y))
                     )
 
     im.save(filename)
 
 
 if __name__ == '__main__':
-    # import doctest
-    # doctest.testmod()
-
     # Create some computational art!
-    generate_art("myart2.png")
-
-    # Test that PIL is installed correctly
-    # test_image("noise.png")
+    generate_art("myart8.png")
```

Comment:
(Possible extension: either using the `os` package to check if various filenames exist, so you don't need to hard-code in a name, or using additional parameters from the command line to set names dynamically (e.g., running `python3 recursive_art.py "myart9.png"`).